custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -117,24 +117,17 @@ def get_custom_url(cls, domain):
 
     def allowed_to_forward(self, episode_case):
         allowed_case_types_and_users = self._allowed_case_type(episode_case) and self._allowed_user(episode_case)
+        if not allowed_case_types_and_users:
+            return False
+
         episode_case_properties = episode_case.dynamic_case_properties()
-        return allowed_case_types_and_users and (
+        return (
             episode_case_properties.get('nikshay_registered', 'false') == 'true' and
             episode_case_properties.get('nikshay_id', False) and
             case_properties_changed(episode_case, [TREATMENT_OUTCOME]) and
-            episode_case_properties.get(TREATMENT_OUTCOME) in treatment_outcome.keys()
-        )
-
-
-def test_submission(person_case):
-    try:
-        phi_location = SQLLocation.objects.get(location_id=person_case.owner_id)
-    except SQLLocation.DoesNotExist:
-        raise NikshayLocationNotFound(
-            "Location with id {location_id} not found. This is the owner for person with id: {person_id}"
-            .format(location_id=person_case.owner_id, person_id=person_case.case_id)
+            episode_case_properties.get(TREATMENT_OUTCOME) in treatment_outcome.keys() and
+            not is_submission_from_test_location(get_person_case_from_episode(episode_case.domain, episode_case))
```

Comment:
@proteusvacuum Can you please add this in a try block to avoid a case where `get_person_case_from_episode` raises a `NikshayCaseNotFound`. It seems that this won't ever happen but good to add since this exception causes phone submissions to break. Have added similar checks to other calls to such methods as well.